server/chat-commands/room-settings.ts

```diff
@@ -933,7 +933,7 @@ export const commands: ChatCommands = {
 		if (room.persist && isGroupchat) return this.errorReply(`This isn't a groupchat.`);
 		if (!room.persist && !isGroupchat) return this.errorReply(`Use /renamegroupchat instead.`);
 		if (isGroupchat) {
-			if (!user.can('lock')) this.checkCan('editroom', null, room);
+			this.checkCan('mute', null, room);
```

Comment:
I suppose this _is_ simpler and usually correct, but I can't say I'm a fan of abusing unrelated permissions in the general case.

Reply:
We could use `Auth#atLeast`, I guess? I think that a `renamegroupchat` permission is a bit specific, personally.